ras_party/views/respondent_view.py

```diff
@@ -91,6 +90,22 @@ def get_respondent_by_email():
     return jsonify(response)
 
 
+@respondent_view.route('/respondents/email', methods=['PATCH'])
```

Comment:
This should be DELETE not PATCH - we can deprecate or remove the old DELETE endpoint if this is the way we're going to do things now

Reply:
Cool... changed to `DELETE` and removed the old delete endpoint